Kickstarter-iOS/Views/Cells/PledgeDescriptionCell.swift

```diff
@@ -46,6 +55,8 @@ final class PledgeDescriptionCell: UITableViewCell, ValueCell {
       |> ksr_constrainViewToEdgesInParent()
 
     self.configureStackView()
+    self.learnMoreButton.addTarget(self, action: #selector(learnMoreButtonTapped), for: .touchUpInside)
```

Comment:
Can we move all this out of the `init` and into a private `configureSubviews` function?

```
  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
    super.init(style: style, reuseIdentifier: reuseIdentifier)

    self.configureSubviews()
    self.bindViewModel()
  }
```